doc/rst/developer/chips/inactive/23.rst

```diff
@@ -760,7 +760,7 @@ the convention of placing complete() as the final statement in the
 body; this will simplify a process of finding these cases and
 removing the unnecessary complete().
 
-These updates can be applied to master in any number of stages
+These updates can be applied to main in any number of stages
```

Comment:
Similar comments as the previous, though here using a quoted 'main' seems sufficiently clear to me.